u2-routing-controllers/features/echoController.js

```diff
@@ -0,0 +1,29 @@
+const getEcho = (_req, res) => {
+  res.send(`<html><head>
+  <style type="text/css">
+    label, button { display: inline-block; margin-top: 16px; }
+    input, textarea { width: 300px; }
+    textarea { height: 150px; }
+  </style>
+</head><body><form action="/echo" method="POST">
+  <label for="echo-name">Name: </label><br />
+  <input id="echo-name" name="name" /><br />
+  <label for="echo-message">Message: </label><br />
+  <textarea id="echo-message" name="message"></textarea><br />
+  <button type="submit">Submit</button>
+</form></body></html>`)
```

Comment:
อูรจ์; ไม่สวย! คุณจะเห็นวิธีที่ดีกว่าใน U3 :)
Eww so ugly! You will see a better way in U3 :)